src/httpd_jsonapi.c

```diff
@@ -2903,6 +2909,44 @@ jsonapi_reply_library_playlist_tracks(struct httpd_request *hreq)
   return HTTP_OK;
 }
 
+static int
+jsonapi_reply_library_playlist_from_queue(struct httpd_request *hreq)
+{
+  const char *param;
+  int ret = 0;
+  char buf[PATH_MAX+15];
```

Comment:
Why `+15`?

Reply:
this was accomodate for the virtual path prefix (`/file:`) - readjusted for the actual length